guides/manual-installation.md

```diff
@@ -0,0 +1,279 @@
+not be able to collect metrics from prometheus and the integrated graphs will not work.
+
+#### Option 3: Install prometheus using the unsupported PackageHub repository
+
+Enable PackageHub repository:
+
+```bash
+SUSEConnect --product PackageHub/15.5/x86_64
+```
+
+Add the prometheus user/group:
+
+```bash
+groupadd --system prometheus
+useradd -s /sbin/nologin --system -g prometheus prometheus
+```
+
+After a successful registration, you can install prometheus using zypper:
+NOTE: some of these steps will trigger messages aboute additional support contract being required.
+
+```bash
+zypper in golang-github-prometheus-prometheus
+```
+
+We will be prompted that a missing dependency can't be satisfied:
+
+```bash
+Problem: nothing provides 'group(prometheus)' needed by the to be installed golang-github-prometheus-prometheus-2.37.6-150100.4.17.1.x86_64
+ Solution 1: do not install golang-github-prometheus-prometheus-2.37.6-150100.4.17.1.x86_64
+ Solution 2: break golang-github-prometheus-prometheus-2.37.6-150100.4.17.1.x86_64 by ignoring some of its dependencies
+```
+
+As we have added the prometheus user/group, we can safely ignore this warning and proceed with the installation (option 2).
+
+Enable and start the prometheus service:
+
+```bash
+systemctl enable --now prometheus
+```
+
+Allow prometheus to be accessible from docker. Add an exception on firewalld:
+
+```bash
+firewall-cmd --zone=public --add-port=9090/tcp --permanent
+firewall-cmd --reload
+```
+
+### Install grafana (Optional)
+
+Enable PackageHub if you didn't enable it in the previous step:
+
+```bash
+SUSEConnect --product PackageHub/15.5/x86_64
+```
+
+Install grafana:
+
+```bash
+zypper install grafana
+```
+
+Enable and start the service:
+
+```bash
+systemctl enable --now grafana-server
+```
+
+Allow grafana to be accessible from docker. Add an exception on firewalld:
+
+```bash
+firewall-cmd --zone=public --add-port=3000/tcp --permanent
+firewall-cmd --reload
+```
+
+Check grafana web UI on port 3000 and set a new password for the user `admin` (we set it to `trento` for this example)
+
+### Install postgresql
+
+```bash
+zypper in postgresql-server
+```
+
+Enable and start the postgresql service:
+
+```bash
+systemctl enable --now postgresql
+```
+
+Initialize the postgresql databases (be sure to update the passwords accordingly):
+
+```bash
+su postgres
+psql
+CREATE DATABASE wanda;
+CREATE USER wanda_user WITH PASSWORD 'wanda-password';
+GRANT ALL PRIVILEGES ON DATABASE wanda TO wanda_user;
+CREATE DATABASE trento;
+CREATE USER trento_user WITH PASSWORD 'web-password';
+GRANT ALL PRIVILEGES ON DATABASE trento TO trento_user;
+CREATE DATABASE trento_event_store;
+GRANT ALL PRIVILEGES ON DATABASE trento_event_store TO trento_user;
+\q
+```
+
+Allow wandadb to be accessible by the docker containers,
+by adding the following line to `pg_hba.conf`:
```

Comment:
Maybe add a path for the user to be able to directly edit it ? without researching? 
on my side it was in `/var/lib/pgsql/data/pg_hba.conf`